open-api/swagger.yaml

```diff
@@ -3125,32 +3174,56 @@ definitions:
     type: object
     properties:
       numericalValueAggregatesPerConcept:
+        $ref: '#/definitions/NumericalAggregates'
+      categoricalValueAggregatesPerConcept:
+        $ref: '#/definitions/NumericalAggregates'
```

Comment:
seems like it should be ` $ref: '#/definitions/CategoricalAggregates'` instead

Reply:
nice catch. thanks!